index.bs

```diff
@@ -5494,30 +5494,30 @@ Methods</h4>
 		this method is called, a {{NotSupportedError}} MUST be
 		thrown.
 
-		Describes which direction the listener is pointing in the 3D
-		cartesian coordinate space. Both a <b>front</b> vector and an
-		<b>up</b> vector are provided. In simple human terms, the
-		<b>front</b> vector represents which direction the person's
-		nose is pointing. The <b>up</b> vector represents the direction
+		{{AudioListener/setOrientation()}} describes which direction the listener is pointing in the 3D
+		cartesian coordinate space. Both a [=front=] vector and an
+		[=up=] vector are provided. In simple human terms, the
+		<dfn dfn>front</dfn> vector represents which direction the person's
```

Comment:
Or is this the definition?

Reply:
Yeah, this is the definition.  Kind of bad to have a forward definition, but I didn't want to restructure everything, and just made explicit links between uses and the definition.